t/055-subreq-vars.t

```diff
@@ -65,8 +65,8 @@ M(http-subrequest-start) {
 --- request
 GET /lua
 --- response_body_like: 500 Internal Server Error
---- error_log chop
-variable "dog" cannot be assigned a value (maybe you forgot to define it first?)
+--- error_log eval
+qr/variable "(dog|cat)" cannot be assigned a value \(maybe you forgot to define it first\?\)/
```

Comment:
When I use Lua 5.1 from RHEL-6 I get this:
```
t/055-subreq-vars.t .. 17/54 
#   Failed test 'TEST 1: set non-existent variables via "vars" option - pattern "variable "dog" cannot be assigned a value (maybe you forgot to define it first?)" matches a line in error.log (req 0)'
#   at lib/Test/Nginx/Socket.pm line 1105.

#   Failed test 'TEST 1: set non-existent variables via "vars" option - pattern "variable "dog" cannot be assigned a value (maybe you forgot to define it first?)" matches a line in error.log (req 1)'
#   at lib/Test/Nginx/Socket.pm line 1105.
```
If I modify test to have "cat" it works. I'll look into this some more.